test/integration/participant_test.rb

```diff
@@ -4,7 +4,7 @@ class ParticipantTest < ActiveSupport::TestCase
 
   context "With a refreshed ldap cache, " do
     setup do
-      @participant = FactoryGirl.create(:participant, :andrewid => "meribyte")
+      @participant = FactoryGirl.create(:participant, :andrewid => "erwilson")
```

Comment:
I'd rather not change this file.  As students leave their info is removed from LDAP but we know Meg's info will always be there.